srcpkgs/residualvm/template

```diff
@@ -17,22 +17,23 @@ distfiles="http://www.residualvm.org/downloads/release/${version}/residualvm-${v
 checksum=f50c83bbc55a8121eefc279e83982b6ec590e608e145b7f750006619dd0bf9e9
 
 build_options="sndio"
+build_options_default="sndio"
```

Comment:
As this is the only build option, maybe remove the option altogether and just enable it?

Reply:
All other packages have sndio as an option, too. It's enabled by default but it's still possible to get an sndio free system.

Reply:
I’m ok with either way. :)